packages/web-components/scripts/generate-tokens.cjs

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
```

Comment:
nit (and this can 100% be done later): convert this file to ESM like `clean.js`.